extensions/amp-autocomplete/validator-amp-autocomplete.protoascii

```diff
@@ -81,6 +92,50 @@ tags: {  # <amp-autocomplete>
     supported_layouts: CONTAINER
   }
 }
+# AMP4EMAIL disallows mustache in src attribute, filter attribute.
+tags: {  # <amp-list>
+  html_format: AMP4EMAIL
+  tag_name: "AMP-AUTOCOMPLETE"
+  spec_name: "AMP-AUTOCOMPLETE (AMP4EMAIL)"
+  requires_extension: "amp-autocomplete"
+  disallowed_ancestor: "AMP-AUTOCOMPLETE"
+  disallowed_ancestor: "AMP-STATE"
+  disallowed_ancestor: "TEMPLATE"
+  attrs: { name: "highlight-user-entry" }
+  attrs: { name: "inline" }
+  attrs: { name: "items" }
+  attrs: { name: "max-entries" }
+  attrs: { name: "min-characters" }
+  attrs: {
+    name: "query"
+    trigger: {
+      also_requires_attr: "src"
+    }
+  }
+  attrs: { name: "submit-on-enter" }
+  attrs: {
+    name: "suggest-first"
+  }
+  attrs: { name: "template" }
```

Comment:
What's the difference between this and the attribute spec on line 130?

Reply:
Thanks for catching this. L130 specifies that the value for the `template` attribute should be one of a set of `TEMPLATE_IDS` which is constructed in the [template validation.](https://github.com/ampproject/amphtml/blob/47a4915b343430e00b6ea7dd8ead98b92d68ddfe/extensions/amp-mustache/validator-amp-mustache.protoascii#L82)